packages/ui/src/components/vuestic-components/va-icon/VaIcon.vdemo.vue

```diff
@@ -1,25 +1,25 @@
 <template>
   <VbDemo>
     <VbCard title="MD">
-      <va-icon class="material-icons">
+      <i class="material-icons">
         home
-      </va-icon>
+      </i>
       <va-icon name="home" />
     </VbCard>
     <VbCard title="FA 4">
       <va-icon name="home" />
```

Comment:
`<va-icon class="fa fa-home" />` - this defaults to md icons. Shouldn't be.

We want to prevent config resolution if there is no name at all.